apps/nextjs-website/src/components/Header.tsx

```diff
@@ -37,20 +62,28 @@ const Header = (props: HeaderWithNavigation) => {
       sx={{
         nav: {
           '.MuiStack-root': {
+            borderColor: 'primary.main',
             a: {
               display: 'flex',
               justifyContent: 'left',
+              '&.MuiTypography-body1': {
+                color: 'text.secondary',
+              },
             },
-            '.MuiStack-root': {
-              // Popup on hover
+            '.MuiTypography-root': {
+              textDecoration: 'none',
+              fontWeight: 600,
+              fontSize: '1rem',
+            },
+            '& .MuiStack-root': {
               zIndex: 10,
+              boxShadow: theme.custom.boxShadow,
```

Comment:
```suggestion
              boxShadow: 'custom.boxShadow',
```
Use theme-aware values.